submissions/Nemooochka/friends-app/js/app.js

```diff
@@ -0,0 +1,35 @@
+import {Cards} from "./cards.js";
+import {Url} from "./url.js";
+import {Filters} from "./filters.js";
+
+const cardsWrapBlock = document.querySelector('.wrap-cards');
+
+const urlData = 'https://randomuser.me/api/?results=20';
+let users;
```

Comment:
Why do you use `let` for declaring this variable?

Reply:
Because later in the code will add info to this variable (get users' info by fetch).